text/0072-kafka-schema-registry.md

```diff
@@ -0,0 +1,111 @@
+- Start Date: 2023-02-01
+- RFC Type: feature
+- RFC PR: n/a
+- RFC Status: draft
+
+# Summary
+
+This RFC proposes introducing a centralized schema repository for the Kafka topics in use at Sentry. It provides mappings between the Kafka topics we are using to the encoding format and schemas for messages on that topic.
+
+Topic registration and schema validation of messages will be optional for the foreseeable future.
+
+# Motivation
+
+Kafka is increasingly used as the message bus between services at Sentry. A Kafka topic and its’ schema not owned by any one service, but is part of the contract between services.
```

Comment:
On ownership. I think we can have owners for schemas. The issue we have is that the contract is not stated anywhere, not that there never is an owner.

Reply:
Reworded to `internal` - the point I was attempting to make is that it's part of the external contract. Hopefully that is clearer.